client/src/helpers/gradientHelper.js

```diff
@@ -110,11 +96,11 @@ function hexToRgb(hex) {
     return [r, g, b];
 }
 
-function getComplementaryColor(rgb) {
-    // Calculate complementary color, then normalize the RGB values
-    return rgb.map(c => (255 - c) / 255);
-}
-
 export function rgbaObjectToNormalizedRgb(rgba) {
     return [rgba.r / 255, rgba.g / 255, rgba.b / 255];
+}
+
+export function hexToNormalizedRGBA(hex) {
+    const rgbArray = hexToRgb(hex)
+    return rgbaObjectToNormalizedRgb({r: rgbArray[0], g: rgbArray[1], b: rgbArray[2]})
```

Comment:
It's not important, but it's nice to use sometimes the destrucutration of js :D (purely personal taste)

```javascript
const [r, g, b] = hexToRgb(hex)
return rgbaObjectToNormalizedRgb({r, g, b})
```

Reply:
Thanks. Looks nicer indeed